lib/flay.rb

```diff
@@ -63,7 +63,7 @@ def self.parse_options args = ARGV
 
     OptionParser.new do |opts|
       opts.banner  = "flay [options] files_or_dirs"
-      opts.version = Flay::VERSION
+      opts.version = VERSION
```

Comment:
Yeah. I get that... theoretically... but is Flay::VERSION no longer relevant? You are subclassing it so I figure you want to report it _somewhere_.

What does your subclass do?